stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Extensions.kt

```diff
@@ -28,28 +28,44 @@ public fun Channel.getUnreadMessagesCount(forUserId: String = ""): Int {
     }
 }
 
+@Deprecated(
+    message = "Use class member instead",
+    level = DeprecationLevel.WARNING,
+)
```

Comment:
We can make this an `ERROR` immediately, as the member property will always shadow the extension of the same name. With the member being added, all of the extension imports will simply become unused, and the migration is just to remove them.

Reply:
makes sense. I'll change it after the release